test/nodejs/route.test.js

```diff
@@ -0,0 +1,31 @@
+const test = require('tape');
+var Valhalla = require('../../');
+
+var config = require('./fixtures/basic_config');
```

Comment:
hm. I guess this is an answer to the question about default configs. what is `node_example.js` used for?

Reply:
it was before we had tests, to play with the binding - I will remove, thanks for flagging!